runtime/nls/cfre/cfrerr.nls

```diff
@@ -1590,3 +1590,98 @@ J9NLS_CFR_ERR_PREVIEW_VERSION2.explanation=Classfiles with minor version 65535 m
+J9NLS_CFR_ERR_ENCLOSING_METHOD_CLASS_INDEX_IS_REFERENCETYPE.user_response=Contact the provider of the classfile for a corrected version.
+
+# ReferenceType is not translatable
+# CONSTANT_Class_info is not translatable
+# host_class_index is not translatable
+J9NLS_CFR_ERR_NEST_HOST_INVALID_REFERENCETYPE_DESCRIPTOR=The CONSTANT_Class_info at the host_class_index cannot be a ReferenceType descriptor
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_NEST_HOST_INVALID_REFERENCETYPE_DESCRIPTOR.explanation=Please consult the Java Virtual Machine Specification for a detailed explanation.
+J9NLS_CFR_ERR_NEST_HOST_INVALID_REFERENCETYPE_DESCRIPTOR.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_NEST_HOST_INVALID_REFERENCETYPE_DESCRIPTOR.user_response=Contact the provider of the classfile for a corrected version.
+
+# ReferenceType is not translatable
+# new is not translatable
+J9NLS_CFR_ERR_NEW_INVALID_REFERENCETYPE_DESCRIPTOR=The new instruction cannot operate on a ReferenceType descriptor
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_NEW_INVALID_REFERENCETYPE_DESCRIPTOR.explanation=Please consult the Java Virtual Machine Specification for a detailed explanation.
+J9NLS_CFR_ERR_NEW_INVALID_REFERENCETYPE_DESCRIPTOR.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_NEW_INVALID_REFERENCETYPE_DESCRIPTOR.user_response=Contact the provider of the classfile for a corrected version.
+
+# aconst_init is not translatable
+J9NLS_CFR_ERR_ACONST_INIT_INVALID_ARRAY=The aconst_init instruction cannot operate on an array
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_ACONST_INIT_INVALID_ARRAY.explanation=Please consult the Java Virtual Machine Specification for a detailed explanation.
+J9NLS_CFR_ERR_ACONST_INIT_INVALID_ARRAY.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_ACONST_INIT_INVALID_ARRAY.user_response=Contact the provider of the classfile for a corrected version.
+
+# ReferenceType is not translatable
+J9NLS_CFR_ERR_OUTER_CLASS_REFERENCETYPE_DESCRIPTOR=Outer class is a ReferenceType descriptor in an InnerClasses attribute
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_OUTER_CLASS_REFERENCETYPE_DESCRIPTOR.explanation=Please consult the Java Virtual Machine Specification for a detailed explanation.
+J9NLS_CFR_ERR_OUTER_CLASS_REFERENCETYPE_DESCRIPTOR.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_OUTER_CLASS_REFERENCETYPE_DESCRIPTOR.user_response=Contact the provider of the classfile for a corrected version.
+# END NON-TRANSLATABLE
+
+# ReferenceType is not translatable
+NEST_MEMBER_INVALID_REFERENCETYPE_DESCRIPTOR=Nest member classes cannot be ReferenceType descriptors
```

Comment:
Should this not be `J9NLS_CFR_ERR_NEST_HOST_INVALID_REFERENCETYPE_DESCRIPTOR` for consistency with other keys and the auxiliary information below?

Reply:
Good point, thanks for the PR